src/main/frontend/CSS/Admin.css

```diff
@@ -1,46 +1,177 @@
-    body{
-        margin: 0;
-    }
+/* Global Reset */
```

Comment:
if it is a global reset should you not do 
`* {
    margin: 0;
    padding: 0;
}`